Its.Log.Monitoring/Telemetry.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Its.Log.Instrumentation;
+
+namespace Its.Log.Monitoring
+{
+    public class Telemetry
+    {
+        private IDictionary<string, object> properties;
+
+        /// <summary>
+        ///     Initializes the <see cref="Instrumentation.Extensions.Telemetry" /> class.
+        /// </summary>
+        static Telemetry()
+        {
+            Formatter<Instrumentation.Extensions.Telemetry>.RegisterForAllMembers();
+        }
```

Comment:
im going to remove the formatter registration. with my change in approach to how responses are returned this isnt needed.